src/iocore/net/SSLUtils.cc

```diff
@@ -1281,7 +1281,7 @@ SSLMultiCertConfigLoader::init_server_ssl_ctx(CertLoadData const &data, const SS
 }
 
 bool
-SSLMultiCertConfigLoader::_setup_session_cache(SSL_CTX *ctx)
+SSLMultiCertConfigLoader::_setup_session_cache(SSL_CTX * /* ctx */)
```

Comment:
We've been marking these like: `/* ctx ATS_UNUSED */` so they are more grep-able.

Reply:
Ah, thank you. Updated.